### ISSUE TITLE
1.13.2 Issue with the water

### ISSUE THREAD

**Issue by IcemanHHW** [sp614x/optifine#1981]
The water is not displaying correctly with and without the animation turned on. It's both with and without using a resource pack. With the texture pack there are some kind of bars in the water, when not using the texturepack the water is flickering when the animation is turned on. Also on some chunks the water displays different than the rest.

Resourcepack water animation on:
![2018-10-26_21 31 59](https://user-images.githubusercontent.com/20383130/47588850-c2ed7180-d967-11e8-9a5f-fad213e7c0db.png)

Resourcepack water animation off:
![2018-10-26_21 32 40](https://user-images.githubusercontent.com/20383130/47588852-c2ed7180-d967-11e8-9305-1f8bb3b17466.png)

Vanilla water animation on:
![2018-10-26_21 33 20](https://user-images.githubusercontent.com/20383130/47588853-c3860800-d967-11e8-8837-725b4ab7b8c6.png)

Vanilla water animation off:
![2018-10-26_21 34 53](https://user-images.githubusercontent.com/20383130/47588854-c3860800-d967-11e8-9efa-fef8c9830d10.png)

**Comment by IcemanHHW** [comment]
Turning off Antialising fixed the problem. But other problems occured. 
Textures not loading in and flickering orange blocks in the water and green 'spikes' in the water. 

![2018-10-27_14 10 54](https://user-images.githubusercontent.com/20383130/47603933-3b4d4480-d9f3-11e8-82ad-cfcabf8b777d.png)
![2018-10-27_14 11 04](https://user-images.githubusercontent.com/20383130/47603934-3b4d4480-d9f3-11e8-8593-1de3d11eb290.png)
![2018-10-27_14 11 16](https://user-images.githubusercontent.com/20383130/47603935-3be5db00-d9f3-11e8-971e-a8270739a6b4.png)
![2018-10-27_14 12 30](https://user-images.githubusercontent.com/20383130/47603936-3be5db00-d9f3-11e8-9300-b39b8bf2fc5d.png)

**Comment by sp614x** [comment]
Is the resource pack BDCraft (which size)?
Have you tried with a lower resolution of BDCraft?

**Comment by IcemanHHW** [comment]
The resourcepack is BDCraft, size is 256x. Tried it on 128x and the problem still happens. This doesn't happen when not using a resourcepack. Tried it while not using Optfine and it didn't happen then. Not sure if it's a problem in BDCraft or a combination from both.

**Comment by iPollox** [comment]
I'm having the same exact issue. The water "pulsates" in those "water blocks/chunks", it goes brighter and darker every second like an animation. Ive tried playing with ALL the settings that could be the issue and havent been able to pin-point it, but I just recorded a video and "anisotropic filtering" seemed to be the cause, but I thought I had played with that setting before and didnt fix the issue.

Video: https://youtu.be/fdAuptVrhDo
